### PR TITLE
Enable Overriding extension

### PR DESCRIPTION
**Pull request by jrdouceur**

### CODE REVIEW

File: DTDL/v3/DTDL.overriding.v1.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+﻿# Overriding extension -- PREVIEW
+
+**Version 1**
+
+**Usable in DTDL version 3**
+
+Overriding is a DTDL language feature that enables a model property of a Property or a Telemetry to be overridden by a value in an instance of the model.
```
Comment (rido-min) [inline DTDL/v3/DTDL.overriding.v1.md:7]: As the description of the extension mentions the instance of the model, I think the example below show include the instance documents.

File: DTDL/v3/DTDL.overriding.v1.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      "name": "currentTemp",
+      "schema": "double",
+      "unit": "degreeFahrenheit"
+    },
+    {
+      "@type": [ "Property", "ValueAnnotation", "Override" ],
+      "name": "currentTempUnit",
+      "annotates": "currentTemp",
+      "overrides": "unit",
+      "schema": "TemperatureUnit",
```
Comment (rido-min) [inline DTDL/v3/DTDL.overriding.v1.md:63]: I dont understand what is `TemperatureUnit` or where is it defined.

**Review by rido-min — CHANGES_REQUESTED** [review]
add an instance in the example

**Comment by jrdouceur** [comment]
This PR is merely copying files from the DTDL repo into opendigitaltwins-dtdl.  Let's pick up this review in a PR on the DTDL repo: https://github.com/Azure/DTDL/pull/151

**Review by rido-min — APPROVED** [review]

**Review by rido-min — APPROVED** [review]

**Pull request merged**